From: Hiroaki Nakamura <hnakamur@gmail.com>
Date: Thu, 7 Jun 2018 22:46:46 +0900
Subject: Skip Tmain parser-own-fields stdout comparison test

This is a workaround for avoiding the following diff which happens only on the Ubuntu PPA build.

--- /<<BUILDDIR>>/universal-ctags-0+SNAPSHOT20180606/Tmain/parser-own-fields.d/stdout-actual.txt      2018-06-07 13:20:17.380303536 +0000
+++ ./Tmain/parser-own-fields.d/stdout-expected.txt   2018-06-07 05:45:08.000000000 +0000
@@ -1,0 +2,3 @@
+bar  input.unknown   /^protected func bar(n);$/;"    f
+baz  input.unknown   /^private func baz(n,...);$/;"  f
+foo  input.unknown   /^public func foo(n, m);$/;"    f
@@ -2,0 +6,3 @@
+bar  input.unknown   /^protected func bar(n);$/;"    f       signature:(n)
+baz  input.unknown   /^private func baz(n,...);$/;"  f       signature:(n,...)
+foo  input.unknown   /^public func foo(n, m);$/;"    f       signature:(n, m)
@@ -3,0 +10,3 @@
+bar  input.unknown   /^protected func bar(n);$/;"    f       protection:protected
+baz  input.unknown   /^private func baz(n,...);$/;"  f       protection:private
+foo  input.unknown   /^public func foo(n, m);$/;"    f       protection:public
@@ -4,0 +14,3 @@
+bar  input.unknown   /^protected func bar(n);$/;"    f       protection:protected    signature:(n)
+baz  input.unknown   /^private func baz(n,...);$/;"  f       protection:private      signature:(n,...)
+foo  input.unknown   /^public func foo(n, m);$/;"    f       protection:public       signature:(n, m)
---
 Tmain/parser-own-fields.d/stdout-expected.txt | 16 ----------------
 1 file changed, 16 deletions(-)
 delete mode 100644 Tmain/parser-own-fields.d/stdout-expected.txt

diff --git a/Tmain/parser-own-fields.d/stdout-expected.txt b/Tmain/parser-own-fields.d/stdout-expected.txt
deleted file mode 100644
index 77ffecc..0000000
--- a/Tmain/parser-own-fields.d/stdout-expected.txt
+++ /dev/null
@@ -1,16 +0,0 @@
-# disabling fields
-bar	input.unknown	/^protected func bar(n);$/;"	f
-baz	input.unknown	/^private func baz(n,...);$/;"	f
-foo	input.unknown	/^public func foo(n, m);$/;"	f
-# enabling signature only
-bar	input.unknown	/^protected func bar(n);$/;"	f	signature:(n)
-baz	input.unknown	/^private func baz(n,...);$/;"	f	signature:(n,...)
-foo	input.unknown	/^public func foo(n, m);$/;"	f	signature:(n, m)
-# enabling protection only
-bar	input.unknown	/^protected func bar(n);$/;"	f	protection:protected 
-baz	input.unknown	/^private func baz(n,...);$/;"	f	protection:private 
-foo	input.unknown	/^public func foo(n, m);$/;"	f	protection:public 
-# enabling both signature and protection
-bar	input.unknown	/^protected func bar(n);$/;"	f	protection:protected 	signature:(n)
-baz	input.unknown	/^private func baz(n,...);$/;"	f	protection:private 	signature:(n,...)
-foo	input.unknown	/^public func foo(n, m);$/;"	f	protection:public 	signature:(n, m)
